akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingRememberEntitiesPerfSpec.scala

```diff
@@ -1,222 +0,0 @@
-  val third = role("third")
-
-  nodeConfig(third)(ConfigFactory.parseString(s"""
-    akka.cluster.sharding.distributed-data.durable.lmdb {
-      # use same directory when starting new node on third (not used at same time)
-      dir = "$targetDir/sharding-third"
-    }
-    """))
-}
-
-class ClusterShardingRememberEntitiesPerfSpecMultiJvmNode1 extends ClusterShardingRememberEntitiesPerfSpec
-class ClusterShardingRememberEntitiesPerfSpecMultiJvmNode2 extends ClusterShardingRememberEntitiesPerfSpec
-class ClusterShardingRememberEntitiesPerfSpecMultiJvmNode3 extends ClusterShardingRememberEntitiesPerfSpec
-
-abstract class ClusterShardingRememberEntitiesPerfSpec
-    extends MultiNodeClusterShardingSpec(ClusterShardingRememberEntitiesPerfSpecConfig)
-    with ImplicitSender {
-  import ClusterShardingRememberEntitiesPerfSpec._
-  import ClusterShardingRememberEntitiesPerfSpecConfig._
-
-  def startSharding(): Unit = {
-    (1 to 3).foreach { n =>
-      startSharding(
-        system,
-        typeName = s"Entity$n",
-        entityProps = ClusterShardingRememberEntitiesPerfSpec.props(),
-        extractEntityId = extractEntityId,
-        extractShardId = extractShardId)
-    }
-  }
-
-  lazy val region1 = ClusterSharding(system).shardRegion("Entity1")
-  lazy val region2 = ClusterSharding(system).shardRegion("Entity2")
-  lazy val region3 = ClusterSharding(system).shardRegion("Entity3")
-
-  // use 5 for "real" testing
-  private val nrIterations = 2
-  // use 5 for "real" testing
-  private val numberOfMessagesFactor = 1
-
-  s"Cluster sharding with remember entities performance" must {
-
-    "form cluster" in within(20.seconds) {
-      join(first, first)
-
-      startSharding()
-
-      // this will make it run on first
-      runOn(first) {
-        region1 ! 0
-        expectMsg(0)
-        region2 ! 0
-        expectMsg(0)
-        region3 ! 0
-        expectMsg(0)
-      }
-      enterBarrier("allocated-on-first")
-
-      join(second, first)
-      join(third, first)
-
-      within(remaining) {
-        awaitAssert {
-          cluster.state.members.size should ===(3)
-          cluster.state.members.unsorted.map(_.status) should ===(Set(MemberStatus.Up))
-        }
-      }
-
-      enterBarrier("all-up")
-    }
-
-    "test when starting new entity" in {
-      runOn(first) {
-        val numberOfMessages = 200 * numberOfMessagesFactor
-        (1 to nrIterations).foreach { iteration =>
-          val startTime = System.nanoTime()
-          (1 to numberOfMessages).foreach { n =>
-            region1 ! (iteration * 100000 + n)
-          }
-          receiveN(numberOfMessages, 20.seconds)
-          val took = NANOSECONDS.toMillis(System.nanoTime - startTime)
-          val throughput = numberOfMessages * 1000.0 / took
-          println(
-            s"### Test1 with $numberOfMessages took ${(System.nanoTime() - startTime) / 1000 / 1000} ms, " +
-            f"throughput $throughput%,.0f msg/s")
-        }
```

Comment:
Are we not interested in both a latency and a throughput bench spec?

Reply:
This still does throughput, just also records latency as it goes